templates/kube-apiserver.yaml

```diff
@@ -29,6 +29,15 @@ spec:
         - --authorization-policy-file=/etc/auth/policies.jsonl
         - --authorization-rbac-super-user=kube-admin
         - --audit-log-path=/dev/stdout
+        - --audit-log-maxsize=50
+        - --audit-log-maxbackup=0
+      livenessProbe:
+        httpGet:
+          host: 127.0.0.1
+          port: 8080
+          path: /healthz
+        initialDelaySeconds: 15
+        timeoutSeconds: 15
```

Comment:
Unless /healthz returns 200 just before 15 seconds timeout, I don't think this will be a problem.
what values would you recommend for timeoutSeconds & periodSeconds?